src/test/java/be/seeseemelk/mockbukkit/WorldMockTest.java

```diff
@@ -867,4 +868,49 @@ void spawnFriendly_Peaceful()
 		assertFalse(armorStand.isDead());
 	}
 
+	@Test
+	void testGetAllowAnimals()
+	{
+		WorldMock world = new WorldMock(Material.DIRT, 3);
+		assertTrue(world.getAllowAnimals());
+	}
+
+	@Test
+	void testGetAllowMonsters()
+	{
+		WorldMock world = new WorldMock(Material.DIRT, 3);
+		assertTrue(world.getAllowMonsters());
+	}
+
+	@Test
+	void testSetSpawnFlags()
+	{
+		WorldMock world = new WorldMock(Material.DIRT, 3);
+		world.setSpawnFlags(false, false);
+		assertFalse(world.getAllowAnimals());
+		assertFalse(world.getAllowMonsters());
```

Comment:
```suggestion
		world.setSpawnFlags(false, true);
		assertFalse(world.getAllowAnimals());
		assertTrue(world.getAllowMonsters());

		world.setSpawnFlags(true, false);
		assertTrue(world.getAllowAnimals());
		assertFalse(world.getAllowMonsters());
```
Just to make sure the correct fields are being updated